.env

```
@@ -0,0 +1,7 @@
+# Example .env file for Blender development
+# Set the full path to Blender's python.exe and blender.exe
+# Edit these paths to match your Blender installation
+BLENDER_PYTHON=D:\Program Files\Blender Foundation\Blender 4.5\4.5\python\bin\python.exe
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)